build/scripts/build_digests_bundle.sh

```diff
@@ -125,10 +144,17 @@ if [ "$DEBUG" == true ]; then
   cat "${PROCESSED_DIR}/bundle.Dockerfile" | sed 's|^|        |g'
 fi
 
-info "Building bundle $PUSH_IMAGE"
-$PODMAN build -t "$PUSH_IMAGE" -f "${PROCESSED_DIR}/bundle.Dockerfile" "$PROCESSED_DIR" | sed 's|^|        |g'
-info "Pushing bundle $PUSH_IMAGE"
-$PODMAN push "$PUSH_IMAGE" 2>&1 | sed 's|^|        |g'
+if [ "$MULTI_ARCH" == "true" ]; then
+  info "Building and pushing bundle $PUSH_IMAGE"
+  docker buildx build -t "$PUSH_IMAGE"  -f "${PROCESSED_DIR}/bundle.Dockerfile" "$PROCESSED_DIR" \
+  --platform "$ARCHITECTURES" \
+  --push | sed 's|^|        |g'
```

Comment:
If you want to indent this output like for normal builds, we should probably also redirect stderr to stdout (otherwise those messages are not indented).
```suggestion
  --push 2>&1 | sed 's|^|        |g'
```